src/app/public/modules/fluid-grid/fluid-grid.component.scss

```diff
@@ -1,5 +1,67 @@
 @import "scss/variables";
 
-.sky-fluid-grid {
-  padding: 0 $sky-fluid-grid-gutter;
+.sky-fluid-grid-gutter-size-small {
```

Comment:
You could use a SCSS mixin for code efficiency (looks like all three sizes use the same styles). See an example, here: https://github.com/blackbaud/skyux-layout/blob/master/src/app/public/modules/fluid-grid/column.component.scss#L3-L9

Reply:
Good idea. This is done, but I had to utilize maps to get the dynamic variable names to work out.